### ISSUE TITLE
Cooldown arg issue

### ISSUE THREAD

**Issue by BahamutxD** [bhhandley/CleveRoidMacros#3]
Trying to make a macro that casts Arcane Rupture, then spams Arcane Missiles until Arcane Rupture is up again.

Arcane Rupture has 15 secs CD but it can reset if you get Temporal Convergence Proc.

The macro seems to ignote/not check its cooldown when the proc happens and keeps spamming Arcane Missiles until the original cd of 15 seconds had passed.

Example macro for reference

`/cast [myrawpower:>818 cooldown:Arcane_Rupture] Arcane Missiles; [myrawpower:>390 nocooldown:Arcane_Rupture] Arcane Rupture`

**Comment by bhhandley** [comment]
Makes sense.  I wasn't aware that reactive had a reset proc.  I do a lot of caching of things to reduce overhead with constantly needing to evaluate all the macros and conditions on your bars but I can make some adjustments for that.  I'm waiting on a couple changes to Nampower which should remove the need to have spells on your bars and easier checking of them.

Let me look into it and see what I can do for now.  My mage is only 13 so I'll probably need you to test a few different updates.

**Comment by bhhandley** [comment]
Someone else had reported the same issue and I pushed a quick fix for this last week that just bypasses the caching for now which should also have resolved your issue.  I've been tied up with other IRL stuff and forgot to get back to you as well.  Can you confirm that this is fixed or not when you have time?

**Comment by BahamutxD** [comment]
> Someone else had reported the same issue and I pushed a quick fix for this last week that just bypasses the caching for now which should also have resolved your issue. I've been tied up with other IRL stuff and forgot to get back to you as well. Can you confirm that this is fixed or not when you have time?

It is working now yes
